diff --git wine-1.9.20/programs/winhlp32/hlpfile.c wine-1.9.20/programs/winhlp32/hlpfile.c
index 3ade6c1..2e1161e 100644
--- wine-1.9.20/programs/winhlp32/hlpfile.c
+++ wine-1.9.20/programs/winhlp32/hlpfile.c
@@ -29,6 +29,7 @@
 #include "wingdi.h"
 #include "winuser.h"
 #include "winhelp.h"
+#include "winnls.h"
 
 #include "wine/debug.h"
 
@@ -1855,6 +1856,12 @@ static BOOL HLPFILE_ReadFont(HLPFILE* hlpfile)
     unsigned    i, len, idx;
     unsigned    face_num, dscr_num, face_offset, dscr_offset;
     BYTE        flag, family;
+    CHARSETINFO ci;
+
+    if (hlpfile->charset == ANSI_CHARSET) {
+        TranslateCharsetInfo((VOID*)GetACP(), &ci ,TCI_SRCCODEPAGE);
+        hlpfile->charset = ci.ciCharset;
+    }
 
     if (!HLPFILE_FindSubFile(hlpfile, "|FONT", &ref, &end))
     {
