diff --git a/dlls/mountmgr.sys/unixlib.c b/dlls/mountmgr.sys/unixlib.c
index 52a3fce..37bd6cb 100644
--- a/dlls/mountmgr.sys/unixlib.c
+++ b/dlls/mountmgr.sys/unixlib.c
@@ -40,7 +40,7 @@ static char *get_dosdevices_path( const char *dev )
 {
     const char *home = getenv( "HOME" );
     const char *prefix = getenv( "WINEPREFIX" );
-    size_t len = (prefix ? strlen(prefix) : strlen(home) + strlen("/.wine")) + sizeof("/dosdevices/") + strlen(dev);
+    size_t len = (prefix ? strlen(prefix) : strlen("/var/lib/wine")) + sizeof("/dosdevices/") + strlen(dev);
     char *path = malloc( len );
 
     if (path)
diff --git a/dlls/ntdll/unix/loader.c b/dlls/ntdll/unix/loader.c
index 96301b1..0df0f51 100644
--- a/dlls/ntdll/unix/loader.c
+++ b/dlls/ntdll/unix/loader.c
@@ -604,9 +604,7 @@ static void set_config_dir(void)
     }
     else
     {
-        if (!home_dir) fatal_error( "could not determine your home directory\n" );
-        if (home_dir[0] != '/') fatal_error( "the home directory %s is not an absolute path\n", home_dir );
-        config_dir = build_path( home_dir, ".wine" );
+        if (!(config_dir = strdup( "/var/lib/wine" ))) fatal_error( "virtual memory exhausted\n" );
     }
 }
 
diff --git a/libs/wine/config.c b/libs/wine/config.c
index 23c76fd..dbab73e 100644
--- a/libs/wine/config.c
+++ b/libs/wine/config.c
@@ -445,17 +445,7 @@ static void init_paths(void)
     }
     else
     {
-        if (!home) fatal_error( "could not determine your home directory\n" );
-        if (home[0] != '/') fatal_error( "your home directory %s is not an absolute path\n", home );
-        config_dir = xmalloc( strlen(home) + sizeof(server_config_dir) );
-        strcpy( config_dir, home );
-        remove_trailing_slashes( config_dir );
-        strcat( config_dir, server_config_dir );
-        if (stat( config_dir, &st ) == -1)
-        {
-            if (errno == ENOENT) return;  /* will be created later on */
-            fatal_perror( "cannot open %s", config_dir );
-        }
+        if (!(config_dir = strdup( "/var/lib/wine" ))) fatal_error( "virtual memory exhausted\n" );
     }
     if (!S_ISDIR(st.st_mode)) fatal_error( "%s is not a directory\n", config_dir );
     if (st.st_uid != getuid()) fatal_error( "%s is not owned by you\n", config_dir );
diff --git a/server/request.c b/server/request.c
index 7021741..3367636 100644
--- a/server/request.c
+++ b/server/request.c
@@ -629,18 +629,7 @@ static char *create_server_dir( int force )
     }
     else
     {
-        const char *home = getenv( "HOME" );
-        if (!home)
-        {
-            struct passwd *pwd = getpwuid( getuid() );
-            if (pwd) home = pwd->pw_dir;
-        }
-        if (!home) fatal_error( "could not determine your home directory\n" );
-        if (home[0] != '/') fatal_error( "your home directory %s is not an absolute path\n", home );
-        if (!(config_dir = malloc( strlen(home) + sizeof("/.wine") ))) fatal_error( "out of memory\n" );
-        strcpy( config_dir, home );
-        for (p = config_dir + strlen(config_dir); p > config_dir; p--) if (p[-1] != '/') break;
-        strcpy( p, "/.wine" );
+        if (!(config_dir = strdup( "/var/lib/wine" ))) fatal_error( "virtual memory exhausted\n" );
     }
 
     if (chdir( config_dir ) == -1)
