diff --git a/lib/_emerge/actions.py b/lib/_emerge/actions.py
index 46cb027..785aba3 100644
--- a/lib/_emerge/actions.py
+++ b/lib/_emerge/actions.py
@@ -2960,7 +2960,7 @@ def run_action(emerge_config):
 			settings=emerge_config.target_config.settings)
 		repo_name_check(emerge_config.trees)
 		repo_name_duplicate_check(emerge_config.trees)
-		config_protect_check(emerge_config.trees)
+		#config_protect_check(emerge_config.trees)
 	check_procfs()
 
 	for mytrees in emerge_config.trees.values():
