diff --git a/Source/WebCore/platform/MIMETypeRegistry.cpp b/Source/WebCore/platform/MIMETypeRegistry.cpp
index f9eabb5f..8aa84679 100644
--- a/Source/WebCore/platform/MIMETypeRegistry.cpp
+++ b/Source/WebCore/platform/MIMETypeRegistry.cpp
@@ -227,6 +227,7 @@ HashSet<String, ASCIICaseInsensitiveHash>& MIMETypeRegistry::supportedNonImageMI
             "text/plain"_s,
             "text/"_s,
             "application/xml"_s,
+            "application/x-fictionbook+xml"_s,
             "application/xhtml+xml"_s,
 #if !PLATFORM(IOS_FAMILY)
             "application/vnd.wap.xhtml+xml"_s,
