diff --git dlls/winex11.drv/keyboard.c dlls/winex11.drv/keyboard.c
index 47a9872..c918d73 100644
--- wine-1.9.20/dlls/winex11.drv/keyboard.c
+++ wine-1.9.20/dlls/winex11.drv/keyboard.c
@@ -1582,7 +1582,7 @@ static HKL get_locale_kbd_layout(void)
      * locale id we return the US one.
      */
 
-    layout = GetUserDefaultLCID();
+    layout = 0x422;
 
     /*
      * Microsoft Office expects this value to be something specific
