diff --git a/dlls/win32u/sysparams.c b/dlls/win32u/sysparams.c
index 4c65bd9..3223969 100644
--- a/dlls/win32u/sysparams.c
+++ b/dlls/win32u/sysparams.c
@@ -2708,7 +2708,7 @@ static UINT_ENTRY( FONTSMOOTHING, 2, DESKTOP_KEY, "FontSmoothing" );
 static UINT_ENTRY( GRIDGRANULARITY, 0, DESKTOP_KEY, "GridGranularity" );
 static UINT_ENTRY( KEYBOARDDELAY, 1, KEYBOARD_KEY, "KeyboardDelay" );
 static UINT_ENTRY( KEYBOARDSPEED, 31, KEYBOARD_KEY, "KeyboardSpeed" );
-static UINT_ENTRY( MENUSHOWDELAY, 400, DESKTOP_KEY, "MenuShowDelay" );
+static UINT_ENTRY( MENUSHOWDELAY, 40, DESKTOP_KEY, "MenuShowDelay" );
 static UINT_ENTRY( MINARRANGE, ARW_HIDE, METRICS_KEY, "MinArrange" );
 static UINT_ENTRY( MINHORZGAP, 0, METRICS_KEY, "MinHorzGap" );
 static UINT_ENTRY( MINVERTGAP, 0, METRICS_KEY, "MinVertGap" );
@@ -2736,27 +2736,27 @@ static BOOL_ENTRY( LOWPOWERACTIVE, FALSE, DESKTOP_KEY, "LowPowerActive" );
 static BOOL_ENTRY( MOUSEBUTTONSWAP, FALSE, MOUSE_KEY, "SwapMouseButtons" );
 static BOOL_ENTRY( POWEROFFACTIVE, FALSE, DESKTOP_KEY, "PowerOffActive" );
 static BOOL_ENTRY( SCREENREADER, FALSE, SCREENREADER_KEY, "On" );
-static BOOL_ENTRY( SCREENSAVEACTIVE, TRUE, DESKTOP_KEY, "ScreenSaveActive" );
+static BOOL_ENTRY( SCREENSAVEACTIVE, FALSE, DESKTOP_KEY, "ScreenSaveActive" );
 static BOOL_ENTRY( SCREENSAVERRUNNING, FALSE, DESKTOP_KEY, "WINE_ScreenSaverRunning" ); /* FIXME - real value */
 static BOOL_ENTRY( SHOWSOUNDS, FALSE, SHOWSOUNDS_KEY, "On" );
 static BOOL_ENTRY( SNAPTODEFBUTTON, FALSE, MOUSE_KEY, "SnapToDefaultButton" );
 static BOOL_ENTRY_MIRROR( ICONTITLEWRAP, TRUE, DESKTOP_KEY, "IconTitleWrap", METRICS_KEY );
 static BOOL_ENTRY( AUDIODESC_ON, FALSE, AUDIODESC_KEY, "On" );
 
-static TWIPS_ENTRY( BORDER, -15, METRICS_KEY, "BorderWidth" );
-static TWIPS_ENTRY( CAPTIONHEIGHT, -270, METRICS_KEY, "CaptionHeight" );
-static TWIPS_ENTRY( CAPTIONWIDTH, -270, METRICS_KEY, "CaptionWidth" );
+static TWIPS_ENTRY( BORDER, 0, METRICS_KEY, "BorderWidth" );
+static TWIPS_ENTRY( CAPTIONHEIGHT, -240, METRICS_KEY, "CaptionHeight" );
+static TWIPS_ENTRY( CAPTIONWIDTH, -240, METRICS_KEY, "CaptionWidth" );
 static TWIPS_ENTRY( ICONHORIZONTALSPACING, -1125, METRICS_KEY, "IconSpacing" );
 static TWIPS_ENTRY( ICONVERTICALSPACING, -1125, METRICS_KEY, "IconVerticalSpacing" );
-static TWIPS_ENTRY( MENUHEIGHT, -270, METRICS_KEY, "MenuHeight" );
-static TWIPS_ENTRY( MENUWIDTH, -270, METRICS_KEY, "MenuWidth" );
+static TWIPS_ENTRY( MENUHEIGHT, -240, METRICS_KEY, "MenuHeight" );
+static TWIPS_ENTRY( MENUWIDTH, -240, METRICS_KEY, "MenuWidth" );
 static TWIPS_ENTRY( PADDEDBORDERWIDTH, 0, METRICS_KEY, "PaddedBorderWidth" );
-static TWIPS_ENTRY( SCROLLHEIGHT, -240, METRICS_KEY, "ScrollHeight" );
-static TWIPS_ENTRY( SCROLLWIDTH, -240, METRICS_KEY, "ScrollWidth" );
+static TWIPS_ENTRY( SCROLLHEIGHT, -195, METRICS_KEY, "ScrollHeight" );
+static TWIPS_ENTRY( SCROLLWIDTH, -195, METRICS_KEY, "ScrollWidth" );
 static TWIPS_ENTRY( SMCAPTIONHEIGHT, -225, METRICS_KEY, "SmCaptionHeight" );
 static TWIPS_ENTRY( SMCAPTIONWIDTH, -225, METRICS_KEY, "SmCaptionWidth" );
 
-static YESNO_ENTRY( BEEP, TRUE, SOUND_KEY, "Beep" );
+static YESNO_ENTRY( BEEP, FALSE, SOUND_KEY, "Beep" );
 
 static DWORD_ENTRY( ACTIVEWINDOWTRACKING, 0, MOUSE_KEY, "ActiveWindowTracking" );
 static DWORD_ENTRY( ACTIVEWNDTRKTIMEOUT, 0, DESKTOP_KEY, "ActiveWndTrackTimeout" );
@@ -2779,7 +2779,7 @@ static PATH_ENTRY( DESKWALLPAPER, DESKTOP_KEY, "Wallpaper" );
 static BYTE user_prefs[8] = { 0x30, 0x00, 0x00, 0x80, 0x12, 0x00, 0x00, 0x00 };
 static BINARY_ENTRY( USERPREFERENCESMASK, user_prefs, DESKTOP_KEY, "UserPreferencesMask" );
 
-static FONT_ENTRY( CAPTIONLOGFONT, FW_BOLD, METRICS_KEY, "CaptionFont" );
+static FONT_ENTRY( CAPTIONLOGFONT, FW_NORMAL, METRICS_KEY, "CaptionFont" );
 static FONT_ENTRY( ICONTITLELOGFONT, FW_NORMAL, METRICS_KEY, "IconFont" );
 static FONT_ENTRY( MENULOGFONT, FW_NORMAL, METRICS_KEY, "MenuFont" );
 static FONT_ENTRY( MESSAGELOGFONT, FW_NORMAL, METRICS_KEY, "MessageFont" );
@@ -2823,25 +2823,25 @@ static struct sysparam_rgb_entry system_colors[] =
 {
 #define RGB_ENTRY(name,val,reg) { { get_rgb_entry, set_rgb_entry, init_rgb_entry, COLORS_KEY, reg }, (val) }
     RGB_ENTRY( COLOR_SCROLLBAR, RGB(212, 208, 200), "Scrollbar" ),
-    RGB_ENTRY( COLOR_BACKGROUND, RGB(58, 110, 165), "Background" ),
-    RGB_ENTRY( COLOR_ACTIVECAPTION, RGB(10, 36, 106), "ActiveTitle" ),
+    RGB_ENTRY( COLOR_BACKGROUND, RGB(64, 64, 64), "Background" ),
+    RGB_ENTRY( COLOR_ACTIVECAPTION, RGB(0, 0, 0), "ActiveTitle" ),
     RGB_ENTRY( COLOR_INACTIVECAPTION, RGB(128, 128, 128), "InactiveTitle" ),
     RGB_ENTRY( COLOR_MENU, RGB(212, 208, 200), "Menu" ),
     RGB_ENTRY( COLOR_WINDOW, RGB(255, 255, 255), "Window" ),
     RGB_ENTRY( COLOR_WINDOWFRAME, RGB(0, 0, 0), "WindowFrame" ),
     RGB_ENTRY( COLOR_MENUTEXT, RGB(0, 0, 0), "MenuText" ),
     RGB_ENTRY( COLOR_WINDOWTEXT, RGB(0, 0, 0), "WindowText" ),
-    RGB_ENTRY( COLOR_CAPTIONTEXT, RGB(255, 255, 255), "TitleText" ),
+    RGB_ENTRY( COLOR_CAPTIONTEXT, RGB(0, 255, 0), "TitleText" ),
     RGB_ENTRY( COLOR_ACTIVEBORDER, RGB(212, 208, 200), "ActiveBorder" ),
     RGB_ENTRY( COLOR_INACTIVEBORDER, RGB(212, 208, 200), "InactiveBorder" ),
-    RGB_ENTRY( COLOR_APPWORKSPACE, RGB(128, 128, 128), "AppWorkSpace" ),
-    RGB_ENTRY( COLOR_HIGHLIGHT, RGB(10, 36, 106), "Hilight" ),
-    RGB_ENTRY( COLOR_HIGHLIGHTTEXT, RGB(255, 255, 255), "HilightText" ),
+    RGB_ENTRY( COLOR_APPWORKSPACE, RGB(64, 64, 64), "AppWorkSpace" ),
+    RGB_ENTRY( COLOR_HIGHLIGHT, RGB(190, 183, 172), "Hilight" ),
+    RGB_ENTRY( COLOR_HIGHLIGHTTEXT, RGB(0, 0, 0), "HilightText" ),
     RGB_ENTRY( COLOR_BTNFACE, RGB(212, 208, 200), "ButtonFace" ),
     RGB_ENTRY( COLOR_BTNSHADOW, RGB(128, 128, 128), "ButtonShadow" ),
     RGB_ENTRY( COLOR_GRAYTEXT, RGB(128, 128, 128), "GrayText" ),
     RGB_ENTRY( COLOR_BTNTEXT, RGB(0, 0, 0), "ButtonText" ),
-    RGB_ENTRY( COLOR_INACTIVECAPTIONTEXT, RGB(212, 208, 200), "InactiveTitleText" ),
+    RGB_ENTRY( COLOR_INACTIVECAPTIONTEXT, RGB(192, 192, 192), "InactiveTitleText" ),
     RGB_ENTRY( COLOR_BTNHIGHLIGHT, RGB(255, 255, 255), "ButtonHilight" ),
     RGB_ENTRY( COLOR_3DDKSHADOW, RGB(64, 64, 64), "ButtonDkShadow" ),
     RGB_ENTRY( COLOR_3DLIGHT, RGB(212, 208, 200), "ButtonLight" ),
@@ -2849,8 +2849,8 @@ static struct sysparam_rgb_entry system_colors[] =
     RGB_ENTRY( COLOR_INFOBK, RGB(255, 255, 225), "InfoWindow" ),
     RGB_ENTRY( COLOR_ALTERNATEBTNFACE, RGB(181, 181, 181), "ButtonAlternateFace" ),
     RGB_ENTRY( COLOR_HOTLIGHT, RGB(0, 0, 200), "HotTrackingColor" ),
-    RGB_ENTRY( COLOR_GRADIENTACTIVECAPTION, RGB(166, 202, 240), "GradientActiveTitle" ),
-    RGB_ENTRY( COLOR_GRADIENTINACTIVECAPTION, RGB(192, 192, 192), "GradientInactiveTitle" ),
+    RGB_ENTRY( COLOR_GRADIENTACTIVECAPTION, RGB(0, 0, 0), "GradientActiveTitle" ),
+    RGB_ENTRY( COLOR_GRADIENTINACTIVECAPTION, RGB(128, 128, 128), "GradientInactiveTitle" ),
     RGB_ENTRY( COLOR_MENUHILIGHT, RGB(10, 36, 106), "MenuHilight" ),
     RGB_ENTRY( COLOR_MENUBAR, RGB(212, 208, 200), "MenuBar" )
 #undef RGB_ENTRY
