diff --git dlls/gdi32/freetype.c dlls/gdi32/freetype.c
index 5467821..b99f58f 100644
--- wine-1.9.20/dlls/gdi32/freetype.c
+++ wine-1.9.20/dlls/gdi32/freetype.c
@@ -3828,10 +3828,12 @@ static void add_font_list(HKEY hkey, const struct nls_update_font_list *fl, int
 {
     const char *sserif = (dpi <= 108) ? fl->sserif_96 : fl->sserif_120;
 
+/*
     RegSetValueExA(hkey, "Courier", 0, REG_SZ, (const BYTE *)fl->courier, strlen(fl->courier)+1);
     RegSetValueExA(hkey, "MS Serif", 0, REG_SZ, (const BYTE *)fl->serif, strlen(fl->serif)+1);
     RegSetValueExA(hkey, "MS Sans Serif", 0, REG_SZ, (const BYTE *)sserif, strlen(sserif)+1);
     RegSetValueExA(hkey, "Small Fonts", 0, REG_SZ, (const BYTE *)fl->small, strlen(fl->small)+1);
+*/
 }
 
 static void set_value_key(HKEY hkey, const char *name, const char *value)
@@ -4035,6 +4037,7 @@ static void update_font_info(void)
             {
                 RegSetValueExA(hkey, "MS Shell Dlg", 0, REG_SZ, (const BYTE *)nls_update_font_list[i].shelldlg,
                                strlen(nls_update_font_list[i].shelldlg)+1);
+/*
                 RegSetValueExA(hkey, "Tms Rmn", 0, REG_SZ, (const BYTE *)nls_update_font_list[i].tmsrmn,
                                strlen(nls_update_font_list[i].tmsrmn)+1);
 
@@ -4050,7 +4053,7 @@ static void update_font_info(void)
                 set_value_key(hkey, nls_update_font_list[i].arial_0.from, nls_update_font_list[i].arial_0.to);
                 set_value_key(hkey, nls_update_font_list[i].courier_new_0.from, nls_update_font_list[i].courier_new_0.to);
                 set_value_key(hkey, nls_update_font_list[i].times_new_roman_0.from, nls_update_font_list[i].times_new_roman_0.to);
-
+*/
                 RegCloseKey(hkey);
             }
             done = TRUE;
diff --git dlls/msi/dialog.c dlls/msi/dialog.c
index fc9c24c..db7700f 100644
--- wine-1.9.20/dlls/msi/dialog.c
+++ wine-1.9.20/dlls/msi/dialog.c
@@ -3512,7 +3512,7 @@ static UINT msi_dialog_reset( msi_dialog *dialog )
 static INT msi_dialog_get_sans_serif_height( HWND hwnd )
 {
     static const WCHAR szSansSerif[] = {
-        'M','S',' ','S','a','n','s',' ','S','e','r','i','f',0 };
+        'M','i','c','r','o','s','o','f','t',' ','S','a','n','s',' ','S','e','r','i','f',0 };
     LOGFONTW lf;
     TEXTMETRICW tm;
     BOOL r;
diff --git loader/wine.inf.in loader/wine.inf.in
index 06ea5b4..bfdc253 100644
--- wine-1.9.20/loader/wine.inf.in
+++ wine-1.9.20/loader/wine.inf.in
@@ -575,25 +575,8 @@ HKLM,%Control%\Session Manager\Memory Management,PagingFiles,,"%24%\pagefile.sys
 HKLM,%Control%\Session Manager\Memory Management,WriteWatch,0x00040002,1
 
 [Fonts]
-HKLM,%FontSubStr%,"Arial Baltic,186",,"Arial,186"
-HKLM,%FontSubStr%,"Arial CE,238",,"Arial,238"
-HKLM,%FontSubStr%,"Arial CYR,204",,"Arial,204"
-HKLM,%FontSubStr%,"Arial Greek,161",,"Arial,161"
-HKLM,%FontSubStr%,"Arial TUR,162",,"Arial,162"
-HKLM,%FontSubStr%,"Courier New Baltic,186",,"Courier New,186"
-HKLM,%FontSubStr%,"Courier New CE,238",,"Courier New,238"
-HKLM,%FontSubStr%,"Courier New CYR,204",,"Courier New,204"
-HKLM,%FontSubStr%,"Courier New Greek,161",,"Courier New,161"
-HKLM,%FontSubStr%,"Courier New TUR,162",,"Courier New,162"
-HKLM,%FontSubStr%,"Helv",,"MS Sans Serif"
-HKLM,%FontSubStr%,"Helvetica",,"Arial"
 HKLM,%FontSubStr%,"MS Shell Dlg 2",,"Tahoma"
-HKLM,%FontSubStr%,"Times",,"Times New Roman"
-HKLM,%FontSubStr%,"Times New Roman Baltic,186",,"Times New Roman,186"
-HKLM,%FontSubStr%,"Times New Roman CE,238",,"Times New Roman,238"
-HKLM,%FontSubStr%,"Times New Roman CYR,204",,"Times New Roman,204"
-HKLM,%FontSubStr%,"Times New Roman Greek,161",,"Times New Roman,161"
-HKLM,%FontSubStr%,"Times New Roman TUR,162",,"Times New Roman,162"
+HKLM,%FontSubStr%,"MS Sans Serif",,"Microsoft Sans Serif"
 HKLM,System\CurrentControlSet\Hardware Profiles\Current\Software\Fonts,"LogPixels",0x10003,0x00000060
 
 [MCI]
