diff --git wine-1.9.20/dlls/ole32/storage32.c wine-1.9.20/dlls/ole32/storage32.c
index 2bc9d3d..87c9eb5 100644
--- wine-1.9.20/dlls/ole32/storage32.c
+++ wine-1.9.20/dlls/ole32/storage32.c
@@ -3044,8 +3044,8 @@ static void StorageImpl_SaveFileHeader(
   /*
    * If the block read failed, the file is probably new.
    */
-  if (FAILED(hr))
-  {
+  /*if (FAILED(hr))
+  {*/
     /*
      * Initialize for all unknown fields.
      */
@@ -3055,7 +3055,7 @@ static void StorageImpl_SaveFileHeader(
      * Initialize the magic number.
      */
     memcpy(headerBigBlock, STORAGE_magic, sizeof(STORAGE_magic));
-  }
+  /*}*/
 
   /*
    * Write the information to the header.
