diff --git a/refs.c b/refs.c
index 261fd82..de95fe9 100644
--- a/refs.c
+++ b/refs.c
@@ -562,19 +562,6 @@ void expand_ref_prefix(struct strvec *prefixes, const char *prefix)
 		strvec_pushf(prefixes, *p, len, prefix);
 }
 
-static const char default_branch_name_advice[] = N_(
-"Using '%s' as the name for the initial branch. This default branch name\n"
-"is subject to change. To configure the initial branch name to use in all\n"
-"of your new repositories, which will suppress this warning, call:\n"
-"\n"
-"\tgit config --global init.defaultBranch <name>\n"
-"\n"
-"Names commonly chosen instead of 'master' are 'main', 'trunk' and\n"
-"'development'. The just-created branch can be renamed via this command:\n"
-"\n"
-"\tgit branch -m <name>\n"
-);
-
 char *repo_default_branch_name(struct repository *r, int quiet)
 {
 	const char *config_key = "init.defaultbranch";
@@ -588,9 +575,7 @@ char *repo_default_branch_name(struct repository *r, int quiet)
 		die(_("could not retrieve `%s`"), config_display_key);
 
 	if (!ret) {
-		ret = xstrdup("master");
-		if (!quiet)
-			advise(_(default_branch_name_advice), ret);
+		ret = xstrdup("main");
 	}
 
 	full_ref = xstrfmt("refs/heads/%s", ret);
