diff --git a/sway/config.c b/sway/config.c
index 9ec40367..af271104 100644
--- a/sway/config.c
+++ b/sway/config.c
@@ -206,7 +206,7 @@ static void config_defaults(struct sway_config *config) {
 	if (!(config->floating_scroll_down_cmd = strdup(""))) goto cleanup;
 	if (!(config->floating_scroll_left_cmd = strdup(""))) goto cleanup;
 	if (!(config->floating_scroll_right_cmd = strdup(""))) goto cleanup;
-	config->default_layout = L_NONE;
+	config->default_layout = L_TABBED;
 	config->default_orientation = L_NONE;
 	if (!(config->font = strdup("monospace 10"))) goto cleanup;
 	config->font_height = 17; // height of monospace 10
