diff --git a/src/build.c b/src/build.c
index 63d3096..0e209e6 100644
--- a/src/build.c
+++ b/src/build.c
@@ -813,6 +813,8 @@ static gchar *prepare_run_cmd(GeanyDocument *doc, gchar **working_dir, guint cmd
 	}
 
 	cmd_string = utils_get_locale_from_utf8(cmd_string_utf8);
+	utils_free_pointers(2, cmd_string_utf8, working_dir_utf8, NULL);
+	return cmd_string;
 
 #ifdef HAVE_VTE
 	if (vte_info.have_vte && vc->run_in_vte)
